modules/apps/object/object-test/src/testFunctional/tests/PopulateObjectFormsConfiguration.testcase

```diff
@@ -383,17 +383,17 @@ definition {
 
 		PopulateObjectFormsConfiguration.saveFormObject();
 
-		AssertElementNotPresent(
+		AssertElementNotPresent.AssertElementNotPresent(
 			locator1 = "FormsAdmin#FORMS_NAVIGATION_BAR_ITEM",
 			value1 = "Entries");
 
 		Navigator.gotoBack();
 
-		Click(
+		Click.Click(
```

Comment:
Are these `Click.Click`s correct? I'm not an expert on Poshi tests, but this kind doesn't show up anywhere else in DXP. This might be causing CI failures.

Reply:
Hey @rodrigocunhaa , can you help us with this answer?

Reply:
Yes, this is wrong. @guilhermedcamacho or @marcelabc, this is the right commit that should be cherry picked:
https://github.com/liferay-objects/liferay-portal/pull/265/commits/d983fcc8e1f85b5247bc052c06e2af82f987d2e7